enforcer-parent/enforcer/src/main/resources/Dockerfile

```diff
@@ -14,11 +14,11 @@
 # limitations under the License.
 # -----------------------------------------------------------------------
 
-FROM adoptopenjdk/openjdk11:jre-11.0.14.1_1-alpine
+FROM adoptopenjdk/openjdk11:jre-11.0.16.1_1
```

Comment:
This is a ubuntu image right?

Reply:
Yes. Had to change run command due to that reason.